### ISSUE TITLE
Font awesome calendar icon for p:calendar component is misplaced on IE 11 and Edge

### ISSUE THREAD

**Issue by stochmalm** [adminfaces/admin-theme#208]
##### Issue Overview
I'm using admin-theme (theme only) for my project and I've got a problem with p:calendar component on IE -> calendar icon is misplaced. 

##### Current Behaviour

![image](https://user-images.githubusercontent.com/22012083/45941050-b67a9f80-bfdc-11e8-9a52-4fd4f304add0.png)

##### Expected Behaviour

![image](https://user-images.githubusercontent.com/22012083/45941064-c85c4280-bfdc-11e8-8ed1-f31ff05812b2.png)

##### How to reproduce  
XHTML:
```html
<h:panelGrid columns="2">
                    <label>#{msgs['contactSearch.menu.contactSearch.input.dateFrom']}</label>
                    <p:calendar id="dateFrom"
                                value=""
                                onkeypress="if (event.keyCode === 13) {return false;}"
                                pattern="#{msgs['patterns.dateFormat']}" mask="true"
                                label="test"
                                showOn="button" locale="pl" lang="pl"/>
                    <label>#{msgs['contactSearch.menu.contactSearch.input.dateTo']}</label>
                    <p:calendar id="dateTo"
                                value=""
                                onkeypress="if (event.keyCode === 13) {return false;}"
                                pattern="#{msgs['patterns.dateFormat']}" mask="true"
                                label="test"
                                showOn="button" locale="pl" lang="pl"/>
</h:panelGrid>
```

I created this css hack to solve this:
```css
/**
   IE calendar icon below input fix.
 */
@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
    body .ui-calendar button.ui-datepicker-trigger > span.ui-icon-calendar {
        top: -1.2em !important;
    }
}

@supports (-ms-ime-align:auto) {
    body .ui-calendar button.ui-datepicker-trigger > span.ui-icon-calendar {
        top: -1.2em !important;
    }
}
```
But it ended up looking like this:
![image](https://user-images.githubusercontent.com/22012083/45941264-a0b9aa00-bfdd-11e8-907d-f17ffd3b2b2b.png)

##### Additional Information

* AdminTheme version:  1.0.0-RC18
* PrimeFaces version:  6.2
* Browser version: Microsoft Edge 42.17134.1.0, IE 11.285.17134.0

**Comment by rmpestano** [comment]
Hi, thanks for reporting!

**Comment by rmpestano** [comment]
Hi, are you sure that the issue occours on Edge? I cannot reproduce here:

![calendar-issue-edge](https://user-images.githubusercontent.com/1592273/46909445-e1d01a80-cf08-11e8-9953-a06dd01d9f18.png)


Although on IE 11 I see various minor issues (not only with calendar):

![calendar-issue-ie11](https://user-images.githubusercontent.com/1592273/46909462-3a9fb300-cf09-11e8-9786-28f50fbedae5.png)

**Comment by stochmalm** [comment]
Hi,
well, I'm not sure what caused that issue, but now everything is alright with my project. Only on IE 11 calendar icon was misplaced but I managed to fix that by appling CSS from my first post. Sorry for the confusion and thanks for the great work ;).

**Comment by rmpestano** [comment]
Hi the issues with IE11 should be fixed in 1.0.0-RC19 release. You can remove your css customizations.
